mcnp/mcnp_funcs.cpp

```diff
@@ -476,16 +476,38 @@ void write_lcad_old(std::ofstream &lcadfile)
   int num_surfs = DAG->num_entities( 2 );
 
   int cmat = 0;
-  double crho, cimp = 1.0;
+  double crho = 0;
+  double cimp_n = 0, cimp_p = 0, cimp_e = 0;
+
+  // Detect which importances are used so all cells, including implicit
+  // complement and graveyard, have these importances
+  bool imp_n_needed = false, imp_p_needed = false, imp_e_needed = false;
+  for( int i = 1; i <= num_cells; ++i ) {
+    moab::EntityHandle vol = DAG->entity_by_index( 3, i );
+    if( DAG->has_prop( vol, "imp.n" )) {
+      imp_n_needed = true;
+      cimp_n = 1;
```

Comment:
reuse of the `cimp_*` variables makes it difficult to understand their intent, and thus to review the method.  Here it looks like they serve as default values, but then later they switch to complement values?  After copying the defaults elsewhere?